pages.it/common/bastet.md

```diff
@@ -0,0 +1,33 @@
+# bastet
+
+> Clone di tetris con un AI maligno
+> I tasti sono personalizzabili, i valori predefiniti sono di seguito
```

Comment:
This is not the command description in the main (English) page, why `maligno`?

Reply:
It reflects the description on the man page
http://manpages.org/bastet/6
It's a Tetris clone with an exceptionally evil AI.